governance.md

```diff
@@ -59,260 +70,185 @@ The foundations of Project governance are:
+
+## Steering Committee
+
+The role of the _pandas_ Steering Committee is to coordinate the activities of the
+different Teams and to ensure that different policies and procedures are carried out in
+a consistent manner. The Steering Committee has no routine decision-making authority,
+except as detailed herein, although in exceptional circumstances it may be called upon
+from time to time to make decisions that are in the best interest of The Project as a
+whole. The Steering Committee will itself decide when a circumstance is exceptional.
+The Steering Committee may not override a PDEP.  When the Steering Committee meets to
+discuss an issue, the members of the Steering Committee are responsible for soliciting
+input from members of the relevant Teams.
+
+The Steering Committee may create a working group to consider changes to the governance
+model described in this document, including the creation of new Teams that support the
+goals of The Project. Changes to the governance model will require a voting process and
+approval equivalent to the process described in PDEP-1, with the exception that all
+Stewards are eligible to vote.  
+
+The Steering Committee may appoint temporary working groups to work on issues, such as
+governance, that fall outside the scope of responsibilities for the existing Teams.
+
+The ideal composition of the Steering Committee consists of 5 people.  The Steering
+Committee will be chosen via an approval voting process from a slate of nominees,
+meaning that each person who is eligible to vote may vote for more than one candidate
+from the slate.  All members of each Team, except the Code of Conduct Team, are 
+eligible to vote.  
+
+### Initial Election
+
+The initial slate for the Steering Committee election will consist of 5 or more
+candidates who volunteer to be on the slate of candidates for the initial Steering
+Committee.  For the initial election, the members of the previous _pandas_ core team
+will vote via an approval voting process.  The top 5 candidates receiving votes
+will then become members of the initial Steering Committee. 
+
+### Subsequent Elections  
+
+Subsequent changes to the membership of the Steering Committee will occur towards the
+end of each calendar year.  By October 31 of each year, each member of the Steering
+Committee will be asked if they would like to continue in that role. If any member
+decides to step down from the Steering Committee, new volunteers will be solicited from
+the group of stewards by the Steering Committee.  Those new volunteers, along with any
+current Steering Committee members who wish to remain on the Steering Committee,
+will then be on a
+slate for an election that will occur by November 15 of that year.  For each year after
+the initial election, the current Steering Committee members plus any new volunteers on
+the slate will be elected via approval voting, with the top 5 candidates elected as
+Steering Committee members for the subsequent year.
```

Comment:
How long do we wait for a "unanimous" vote? Historically I think people just kind of fade away from the project. By the letter this might put us in a spot where someone goes into inactive and the rest of the team has to remove them before making further decisions

Reply:
With one year terms, and getting a commitment in October that someone is willing to serve on the Steering Committee in the following year, we're at minimal risk here.  It would be odd for someone to say on October 15 "Yes, I'll continue on the steering committee for the next year" and then become inactive pretty quickly.  Having said that, if it should occur, there is less than a year until the next election, so the position can be filled.

Maybe you're referring to the Teams identifying by unanimous vote how to remove someone from a Team?

Reply:
Yea sorry - this comment should have been in the Teams section. Let me restart it down there for better visibility